### PR TITLE
[FXlab] Adding volume toggle to top Toolbar. Fixes #675

### LINKED ISSUE #675: FXLab can emit loud feedback
**Short Description**

FXLab can emit a loud high pitched tone due to feedback.

**Steps To Reproduce**

Launch FXLab with no hardware attached.

**Expected behavior**
Pleasant UI.

**Actual behavior**
Loud screeching sound that causes panic and angry stares from fellow bus passengers.

Suggest adding a simple automated feedback suppressor of some sort.
https://en.wikipedia.org/wiki/Feedback_suppressor
https://www.rane.com/note158.html

Another option would be a start/stop button so feedback can be killed quickly.

### PR DESCRIPTION
**Pull request by dturner**

### CODE REVIEW

File: apps/fxlab/app/src/main/java/com/mobileer/androidfxlab/MainActivity.kt [hunk 1]
```diff
@@ -42,16 +44,16 @@ import com.mobileer.androidfxlab.datatype.Effect
 class MainActivity : AppCompatActivity() {
 
     private var TAG: String = this.toString()
-
     lateinit var binding: ActivityMainBinding
+	private var isAudioEnabled: Boolean = false
```
Comment (philburk) [inline apps/fxlab/app/src/main/java/com/mobileer/androidfxlab/MainActivity.kt]: tab or 4 spaces

File: apps/fxlab/app/src/main/java/com/mobileer/androidfxlab/MainActivity.kt [hunk 2]
```diff
@@ -162,7 +165,29 @@ class MainActivity : AppCompatActivity() {
         }, Handler())
     }
 
-    @RequiresApi(Build.VERSION_CODES.M)
+	override fun onCreateOptionsMenu(menu: Menu) : Boolean {
```
Comment (philburk) [inline apps/fxlab/app/src/main/java/com/mobileer/androidfxlab/MainActivity.kt]: tab or 4 spaces?
no indentation

File: apps/fxlab/app/src/main/java/com/mobileer/androidfxlab/MainActivity.kt [hunk 3]
```diff
@@ -162,7 +165,29 @@ class MainActivity : AppCompatActivity() {
...
+    	// Inflate the menu; this adds items to the action bar if it is present.
+    	getMenuInflater().inflate(R.menu.toolbar_menu, menu);
+    	return true
+	}
+
+	override fun onOptionsItemSelected(item: MenuItem) = when (item.itemId) {
+		R.id.action_toggle_mute -> {
```
Comment (philburk) [inline apps/fxlab/app/src/main/java/com/mobileer/androidfxlab/MainActivity.kt]: 2 tabs, should be 8 spaces

File: apps/fxlab/app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -22,13 +22,34 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+	<com.google.android.material.appbar.AppBarLayout
+		android:id="@+id/appBar"
```
Comment (philburk) [inline apps/fxlab/app/src/main/res/layout/activity_main.xml]: more tabs!

Comment (dturner) [reply apps/fxlab/app/src/main/res/layout/activity_main.xml]: I know - so annoying!! I have everything in Android Studio checked to use spaces. I don't know how they keep creeping in!

Comment (dturner) [reply apps/fxlab/app/src/main/res/layout/activity_main.xml]: I think they're fixed now.

**Pull request merged**